src/transformers/pipelines/__init__.py

```diff
@@ -25,6 +25,8 @@
 
 from numpy import isin
 
+from transformers import SpeechEncoderDecoderConfig, VisionTextDualEncoderConfig
```

Comment:
Let's not add new dependencies between pipeline and specific models if we can avoid it. Here we can detect the proper model types I believe.

If those are kept, they should be relative imports like the rest.

Reply:
Perfectly understandable. It's done.